### ISSUE TITLE
PasswordBox's MaterialDesignFloatingHintPasswordBox style problem

### ISSUE THREAD

**Issue by Wolfteam** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1665]
Hiho, the MaterialDesignFloatingHintPasswordBox style shows two lines

![passwordboxproblem](https://user-images.githubusercontent.com/1976115/74868873-f3a7a600-5324-11ea-97ae-305727098e8b.PNG)

A workaround for this may be:
````
<PasswordBox.Style>
       <Style BasedOn="{StaticResource MaterialDesignFloatingHintPasswordBox}" TargetType="PasswordBox">
             <Setter Property="materialDesign:TextFieldAssist.DecorationVisibility" Value="Hidden" />
       </Style>
</PasswordBox.Style>
````

**Comment by AmRo045** [comment]
I think this issue fixed by this PR #1637

**Comment by Wolfteam** [comment]
> I think this issue fixed by this PR #1637

You are right, thanks
